include/temoto_robot_manager/robot_manager_interface.h

```diff
@@ -251,6 +270,27 @@ class RobotManagerInterface : public temoto_core::BaseSubsystem
     return pose;
   }
 
+  std::vector<double> getCurrentJointValues(const std::string& robot_name, const std::string& planning_group)
+ {
+    std::vector<double> joints;
+    temoto_robot_manager::RobotGetTarget msg;
+    msg.request.robot_name = robot_name;
+    msg.request.planning_group = planning_group;
+    msg.request.get_current_state = msg.request.JOINT_STATE;
+    if (!client_get_manipulation_target_.call(msg))
+    {
+      throw TEMOTO_ERRSTACK("Unable to reach robot_manager");
+    }
+
+    if (!msg.response.success)
+    {
+      throw TEMOTO_ERRSTACK("Unsuccessful attempt to invoke 'getEndEffPose'");
+    }
+
+    joints = msg.response.joint_values;
```

Comment:
no need to define a local variable `joints`, just return the `msg.response.joint_values`

Reply:
Ok, completely agree